### ISSUE TITLE
How keras preprocess the data in certain way by default

### ISSUE THREAD

**Issue by danieljf24** [fchollet/keras#521]
I built a very simple model to learn keras, while I am in trouble now. The loss returned by keras is small (about 2), which is different form what I expected. 
I implemented this modle with pure theano, and the loos is about 4000.
I wonder if the keras preprocess the data in certain way by default, such as normalization. Thank your very much.

        model = Sequential()

        model.add(Dense(n_in, n_hidden, init='lecun_uniform'))
        model.add(Activation('relu'))
        model.add(Dropout(dropout))

        model.add(Dense(n_hidden, n_out, init='lecun_uniform'))
        model.add(Activation('relu'))

        sgd = SGD(lr=0.1, decay=1e-6, momentum=0.9, nesterov=True)
        model.compile(loss='mean_squared_error', optimizer=sgd)

**Comment by fchollet** [comment]
There is no preprocessing by default whatsoever. Keras avoids doing anything implicitly, especially something as sensitive as data preprocessing.

Worth noting that your output will always be positive (last layer is `relu`), that might be the reason for what you're seeing.
